lighthouse-core/audits/predictive-perf.js

```diff
@@ -53,6 +54,8 @@ class PredictivePerf extends Audit {
     const ttfcpui = await LanternFirstCPUIdle.request({trace, devtoolsLog, settings}, context);
     const si = await LanternSpeedIndex.request({trace, devtoolsLog, settings}, context);
     const eil = await LanternEil.request({trace, devtoolsLog, settings}, context);
+    // TODO: we don't have LCP in the lantern test yet
```

Comment:
move comment to L17?
also can you file an issue that tracks the trace-updating/calibration bit and drop the issue # here?

Reply:
https://github.com/GoogleChrome/lighthouse/issues/9953